lib/index.js

```diff
@@ -94,17 +135,23 @@ _applyPoints.forEach(function (applyPoint) {
 
             } else if (typeof routePolicy === 'function') {
 
+                if (!aggregateApplyPoint && routePolicy.runs && !routePolicy.applyPoint) {
+
```

Comment:
I wonder if we need to make "runs" a bit more descriptive.  Or stick with the "apply" terminology.  thoughts?